ibm/service/power/resource_ibm_pi_volume_bulk.go

```diff
@@ -0,0 +1,537 @@
+				DiffSuppressFunc: flex.ApplyOnce,
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeString,
+				ValidateFunc:     validate.InvokeValidator("ibm_pi_volume", Arg_AffinityPolicy),
+			},
+			Arg_AffinityVolume: {
+				ConflictsWith:    []string{Arg_AffinityInstance},
+				Description:      "Volume (ID or Name) to base volume affinity policy against; required if requesting 'affinity' and 'pi_affinity_instance' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeString,
+			},
+			Arg_AntiAffinityInstances: {
+				ConflictsWith:    []string{Arg_AntiAffinityVolumes},
+				Description:      "List of pvmInstances to base volume anti-affinity policy against; required if requesting 'anti-affinity' and 'pi_anti_affinity_volumes' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				Elem:             &schema.Schema{Type: schema.TypeString},
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeList,
+			},
+			Arg_AntiAffinityVolumes: {
+				ConflictsWith:    []string{Arg_AntiAffinityInstances},
+				Description:      "List of volumes to base volume anti-affinity policy against; required if requesting 'anti-affinity' and 'pi_anti_affinity_instances' is not provided.",
+				DiffSuppressFunc: flex.ApplyOnce,
+				Elem:             &schema.Schema{Type: schema.TypeString},
+				ForceNew:         true,
+				Optional:         true,
+				Type:             schema.TypeList,
+			},
+			Arg_CloudInstanceID: {
+				Description:  "The GUID of the service instance associated with an account.",
+				ForceNew:     true,
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
+			},
+			Arg_Count: {
+				Default:      1,
+				Description:  "Number of volumes to create. Default 1.",
+				ForceNew:     true,
+				Optional:     true,
+				Type:         schema.TypeInt,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+			Arg_ReplicationEnabled: {
+				Computed:    true,
+				Description: "Indicates if the volume should be replication enabled or not.",
+				Optional:    true,
+				Type:        schema.TypeBool,
+			},
+			Arg_ReplicationSites: {
+				Description: "List of replication sites for volume replication.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				ForceNew:    true,
+				Optional:    true,
+				Set:         schema.HashString,
+				Type:        schema.TypeSet,
+			},
+			Arg_UserTags: {
+				Description: "The user tags attached to this resource.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				ForceNew:    true,
```

Comment:
Could you remove the `forceNew` since we can update this.

Reply:
I have all updates disabled on that resource to avoid calling the individual volume update a ton of times which is why force new is everywhere.